### PR TITLE
Update to node 18

### PR DESCRIPTION
**Pull request by pratikyadav**
Update to node 18.

### CODE REVIEW

**Comment by oielbanna** [comment]
Did you get a chance to test this?

File: lib/template.js [hunk 1]
```diff
@@ -3,8 +3,8 @@
 const cf = require('@mapbox/cloudfriend');
 const table = require('@mapbox/watchbot-progress').table;
 const path = require('path');
-const pkg = require(path.resolve(__dirname, '..', 'package.json'));
-const dashboard = require(path.resolve(__dirname, 'dashboard.js'));
+const pkg = require(path.resolve('usr', '../package.json'));
```
Comment (XiaotuZhang) [inline lib/template.js:6]: __dirname is not available anymore? The change replacing to `usr`, does it mean now we are hard coding the directory to `/usr/`?

**Review by XiaotuZhang — COMMENTED** [review]
Can you explain a bit on path resolving?

File: lib/template.js (hunk 1 above)
Comment (pratikyadav) [reply lib/template.js]: Reverted back. I was trying to fix the failing codebuild tests. @mappingvermont helped me out in bumping the runtime of codebuild.

**Comment by pratikyadav** [comment]
> Did you get a chance to test this?

@oielbanna yes this has been tested successfully. 🟢

**Review by oielbanna — APPROVED** [review]
Thanks for your contribution 🚀

**Pull request merged**